docs/data/xrootd/install-standalone.md

```diff
@@ -73,21 +96,19 @@ under `/etc/xrootd/config.d/` as follows:
 
     If you want to serve everything under your configured `rootdir`, you don't have to change anything.
 
-    !!! note
+    !!! danger
         The directories specified this way are writable by default.
         Access controls should be managed via [authorization configuration](#configuring-authorization).
 
 1. In `/etc/xrootd/config.d/10-common-site-local.cfg`, add a line to set the `resourcename` variable to the
-   [resource name](../../common/registration.md#registering-resources) of your XRootD service.
+   [resource name](../../common/registration.md#registering-resources) of your XRootD service or to a value matching
+   your supported VOs policies.
```

Comment:
GH is not letting me make a suggestion for this but how about something like
```
1. In `/etc/xrootd/config.d/10-common-site-local.cfg`, add a line to set the `resourcename` variable.
   Unless your supported VOs' policies state otherwise,
   this should match the [resource name](../../common/registration.md#registering-resources) of your XRootD service.
```

Reply:
Yeah, that's better!